src/functions.rs

```diff
@@ -2919,6 +2919,75 @@ pub fn initialize_terminated<'a>(functions: &'a FunctionMap<'a>) -> (Vec<String>
     (term_ret_vec, nonterm_ret_vec)
 }
 
+// Helper function to substitue class name if needed
+fn cil_name_helper(
```

Comment:
Can this be named something more descriptive than "helper"?  And a clear comment explaining what it does above and beyond a vanilla "get_cil_name()" would be really helpful.